app/screens/channel_members/index.js

```diff
@@ -16,15 +16,17 @@ function makeMapStateToProps() {
     const getChannelMembers = makeGetProfilesInChannel();
 
     return (state) => {
-        const currentChannelId = getCurrentChannelId(state);
+        const currentChannel = getCurrentChannel(state);
         let currentChannelMembers = [];
-        if (currentChannelId) {
-            currentChannelMembers = getChannelMembers(state, currentChannelId, true);
+        if (currentChannel.id) {
+            currentChannelMembers = getChannelMembers(state, currentChannel.id, true);
         }
 
+        const canManageUsers = currentChannel.group_constrained ? false : canManageChannelMembers(state);
```

Comment:
Minor nit, but I think this might be a bit clearer than using a ternary statement
```suggestion
        const canManageUsers = canManageChannelMembers(state) && !currentChannel.group_constrained;
```